organization/planning/plan-of-work-2023.md

```diff
@@ -0,0 +1,215 @@
+
+### Previous Plans
+
+* [Plan of work 2022](./plan-of-work-2022.html)
+* [Plan of work 2021](./plan-of-work-2021.html)
+
+## Overall HSF Activities
+
+### General Meeting Series
+
+* Continue with regular [coordination meetings]({{ site.baseurl
+  }}/meetings/coordination.html) every 2 weeks (on the odd-numbered weeks of the
+  year by default).
+* The [Compute Accelerator Forum]({{ site.baseurl
+  }}/meetings/compute-accelerator-forum.html), co-organised with SIDIS and
+  openlab, has established itself as a strong series of meetings with good turnout,
+  focusing on new compute architectures. We shall continue to find interesting topics to present.
+  * As there is a significant cross-over with the WG activities in many places, we should
+    consider joint organised meetings when appropriate
+* Monthly [*Software and Computing Roundtable*]({{ site.baseurl
+  }}/meetings/roundtable.html) meetings, co-organised with JLab and BNL, cover general topics of interest and the engagement
+  of the HSF increased through 2022. We should continue to advertise the meeting
+  and use this as an engagement opportunity for speaking to Nuclear Physics
+  colleagues.
+
+### Workshops
+
+* As travel restictions have now largely eased, we can organise more workshops
+as in-person events, which we know are extremely productive, after last year's
+[Analysis Ecosystem II meeting](https://indico.cern.ch/event/1125222/)
+* In particular, we plan a [pre-CHEP workshop with WLCG](https://indico.cern.ch/event/1230126/), that should focus on
+common topics between software and computing infrastructure
+  * Use of non-x86 hardware on at WLCG sites
+  * Analysis Facilities
+
+See below for PyHEP and Generators workshop ideas.
+
+## Working Groups
+
+### Physics Event generators
+
+* Priorities:
+  * Event generation:
+    * Event generation for LHC and HL-LHC
+    * Cross-cutting aspects with other experiments in NHEP
+  * MCEG R&D:
+    * Cutting-edge software technology
+    * Standards for shared elements
+* Working group discussions:
+  * More engagement in the WG?
+  * Follow up on LHCC and Snowmass/P5
+  * New documents?
+* Workshop planning:
+  * MC4EIC 2023 (tba)
+  * Follow up of the [2018 workshop](https://indico.cern.ch/event/751693/)?
+* Training:
+  * Tuning tutorial (co-organize with Training WG)
+
+### PyHEP
+
+#### PyHEP 2023
+
+* [The 5th anniversary workshop!](https://indico.cern.ch/event/1252095/)
+* Online workshop, we converged on the week of October 9‒13
+  * To be advertised soon, but see next slide …
+* Targeting as always the broader audience of developers and physicists:
+  * Idea of connecting Python users in HEP
+
+#### PyHEP Dev Workshop
+
+* New this year: [In person workshop for Python package developers](https://indico.cern.ch/event/1234156) @ Princeton, USA
+  * 25-29 July, at Princeton, aiming for ~40 people
+* *Idea to get developers getting their software products to work together as a coherent ecosystem*
```

Comment:
```suggestion
* *Idea to get developers getting their software products to work together as a coherent ecosystem, and plan a coherent roadmap and make priorities for the upcoming year*
```

Reply:
maybe consider a word other than coherent?

Reply:
This is what we wrote on the site. But feel free to suggest another word. Cc @jpivarski.

Maybe "interoperable"?